docs/App.jsx

```diff
@@ -100,6 +116,38 @@ class App extends React.Component {
           defaultValue="0"
         />
         <br />
+        <label>
```

Comment:
I agree with you for demo purposes - I was thinking it was nice from a testing perspective to have the uncontrolled behaviour exercised in the demo.  Happy to remove though! :)